From: Egbert Eich <eich@suse.com>
Date: Tue Jul 23 08:10:36 2013 +0200
Subject: drm/mgag200: Fix logic in mgag200_bo_pin() (v2)
Patch-Mainline: v3.11-rc4
Git-commit: 030f19f0e24c020e3577d9a0da878e417efc2b39
References: 
Signed-off-by: Egbert Eich <eich@suse.com>

add missing 'return 0;'.

v2: Simplify patch as suggested by Dave Airlie <airlied@redhat.com>

Signed-off-by: Egbert Eich <eich@suse.com>
---
 drivers/gpu/drm/mgag200/mgag200_ttm.c |    1 +
 1 file changed, 1 insertion(+)

--- linux-3.0-SLE11-SP3.orig/drivers/gpu/drm/mgag200/mgag200_ttm.c
+++ linux-3.0-SLE11-SP3/drivers/gpu/drm/mgag200/mgag200_ttm.c
@@ -377,6 +377,7 @@ int mgag200_bo_pin(struct mgag200_bo *bo
 		bo->pin_count++;
 		if (gpu_addr)
 			*gpu_addr = mgag200_bo_gpu_offset(bo);
+		return 0;
 	}
 
 	mgag200_ttm_placement(bo, pl_flag);
